tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from inspire_schemas.utils import load_schema
+from inspirehep.dojson.utils import validate
+from inspirehep.modules.workflows.tasks.arxiv import (
+    arxiv_derive_inspire_categories
+)
+
+
+class StubObj(object):
```

Comment:
@jacquerie @david-caro @kaplun I will try to make what @jmartinm is saying for this PR. Anyway it would be nice have a clear policy about how to reuse code in the tests! :smile: